package.json

```diff
@@ -25,95 +25,99 @@
-    "ajv": "^5.2.3",
+    "@types/jasmine": "2.8.2",
+    "@types/jquery": "^3.2.17",
+    "@types/node": "^8.5.2",
+    "ajv": "^5.5.2",
     "angular2-template-loader": "^0.6.2",
-    "autoprefixer": "^6.5.3",
-    "awesome-typescript-loader": "^3.2.1",
+    "autoprefixer": "^7.2.3",
+    "awesome-typescript-loader": "^3.4.1",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
-    "babel-preset-env": "^1.6.0",
-    "codelyzer": "^4.0.0",
+    "babel-preset-env": "^1.6.1",
+    "codelyzer": "^4.0.2",
     "compass-sass-mixins": "^0.12.7",
-    "copy-webpack-plugin": "^4.0.1",
-    "core-js": "^2.4.1",
-    "cross-env": "^5.0.5",
-    "css-loader": "^0.26.1",
+    "copy-webpack-plugin": "^4.3.1",
+    "core-js": "^2.5.3",
+    "cross-env": "^5.1.3",
+    "css-loader": "^0.28.7",
     "cssnano": "^3.10.0",
-    "exports-loader": "^0.6.3",
-    "file-loader": "^0.10.0",
-    "html-loader": "^0.4.4",
-    "html-webpack-plugin": "^2.26.0",
-    "istanbul-instrumenter-loader": "^2.0.0",
-    "jasmine-core": "~2.5.2",
-    "jasmine-spec-reporter": "~3.2.0",
-    "json-loader": "^0.5.4",
-    "karma": "~1.4.1",
-    "karma-chrome-launcher": "~2.0.0",
+    "exports-loader": "^0.6.4",
+    "file-loader": "^1.1.6",
+    "html-loader": "^0.5.1",
+    "html-webpack-plugin": "^2.30.1",
+    "istanbul-instrumenter-loader": "^3.0.0",
+    "jasmine-core": "~2.8.0",
+    "jasmine-spec-reporter": "~4.2.1",
+    "karma": "~2.0.0",
+    "karma-chrome-launcher": "~2.2.0",
     "karma-cli": "~1.0.1",
-    "karma-coverage-istanbul-reporter": "^0.2.0",
-    "karma-jasmine": "~1.1.0",
+    "karma-coverage-istanbul-reporter": "^1.3.3",
+    "karma-jasmine": "~1.1.1",
     "karma-jasmine-html-reporter": "^0.2.2",
-    "karma-sourcemap-loader": "^0.3.7",
-    "less-loader": "^2.2.3",
-    "node-sass": "^4.3.0",
-    "postcss-loader": "^0.13.0",
-    "postcss-url": "^5.1.2",
-    "protractor": "~5.1.0",
+    "less-loader": "^4.0.5",
+    "node-sass": "^4.7.2",
+    "postcss-custom-properties": "^6.2.0",
+    "postcss-load-config": "^1.2.0",
+    "postcss-loader": "^2.0.9",
+    "postcss-url": "^7.3.0",
+    "protractor": "~5.2.2",
     "raw-loader": "^0.5.1",
-    "resolve-url-loader": "^2.1.0",
-    "sass-loader": "^6.0.3",
-    "script-loader": "^0.7.0",
-    "source-map-loader": "^0.1.5",
-    "style-loader": "^0.13.1",
-    "stylus-loader": "^2.4.0",
-    "ts-loader": "2.0.3",
-    "ts-node": "~2.0.0",
+    "resolve-url-loader": "^2.2.1",
+    "sass-loader": "^6.0.6",
+    "script-loader": "^0.7.2",
+    "source-map-loader": "^0.2.3",
+    "style-loader": "^0.19.1",
+    "stylus-loader": "^3.0.1",
+    "ts-loader": "3.2.0",
+    "ts-node": "~4.1.0",
     "tslint": "^5.8.0",
-    "tslint-loader": "^3.3.0",
-    "typescript": "~2.5.3",
-    "typings": "2.1.0",
-    "url-loader": "^0.5.7",
-    "webpack": "~3.6.0",
-    "webpack-dev-server": "~2.9.1",
-    "webpack-node-externals": "^1.6.0"
+    "tslint-loader": "^3.5.3",
+    "typescript": "2.6.2",
```

Comment:
Is it a good idea to upgrade to the latest version of TypeScript which is not officially supported by Angular 5.1? (It is planned for Angular 5.2)